src/main/java/com/palantir/gradle/graal/GraalExtension.java

```diff
@@ -44,7 +44,7 @@
     private static final String DOWNLOAD_BASE_URL_GRAAL_19_3 =
             "https://github.com/graalvm/graalvm-ce-builds/" + "releases/download/";
     private static final String DEFAULT_GRAAL_VERSION = "20.2.0";
-    private static final List<String> SUPPORTED_JAVA_VERSIONS = Arrays.asList("11", "8");
+    private static final List<String> SUPPORTED_JAVA_VERSIONS = Arrays.asList("16", "11", "8");
```

Comment:
I think it would be reasonable to remove this validation entirely, that way it doesn't need to be updated as each new java version is supported by graalvm native-image.

Reply:
I think it's useful to keep the validation in place as users won't typically know which is the latest GraalVM version supporting their target Java version. For example, users could attempt to use Java 16 with GraalVM 21.0.0 and that would fail to download the JDK. Granted, they'll get a download error, but it might make sense to keep it this way for now. If it becomes problematic in the future, we can remove it altogether as the scope of that is a bigger than this PR.